Server/mods/deathmatch/logic/packets/CPlayerJoinDataPacket.cpp

```diff
@@ -26,13 +26,16 @@ bool CPlayerJoinDataPacket::Read(NetBitStreamInterface& BitStream)
     m_bOptionalUpdateInfoRequired = BitStream.ReadBit();
 
     if (BitStream.Read(m_ucGameVersion) && BitStream.ReadStringCharacters(m_strNick, MAX_PLAYER_NICK_LENGTH) &&
-        BitStream.Read(reinterpret_cast<char*>(&m_Password), 16) && BitStream.ReadStringCharacters(m_strSerialUser, MAX_SERIAL_LENGTH))
+        BitStream.Read(reinterpret_cast<char*>(&m_Password), 16) && BitStream.ReadStringCharacters(m_strSerialUser, MAX_SERIAL_LENGTH) &&
+        BitStream.ReadStringCharacters(m_strConnectArgs, MAX_CONNECT_ARGS_LENGTH))
     {
```

Comment:
Does this change breaks old clients?

Reply:
No, because we are checking the bitstream versions on both sides, if the bitstream is not updated, the packet remains the same.

Reply:
Sorry, I thought this was some other code 😅 You're right, I will update this to check player bitstream version